src/main/resources/db/migration/V1_29__populate_metadata_table.sql

```diff
@@ -0,0 +1,159 @@
+('exclusion','other_preferred_method_guide','FALSE','TRUE'),
+('exclusion','same_type_rule_guide','FALSE','TRUE'),
+('exclusion','schedule_frequency_guide','FALSE','TRUE'),
+('flyway_schema_history','installed_rank','FALSE','FALSE'),
+('flyway_schema_history','version','FALSE','FALSE'),
+('flyway_schema_history','description','FALSE','FALSE'),
+('flyway_schema_history','type','FALSE','FALSE'),
+('flyway_schema_history','script','FALSE','FALSE'),
+('flyway_schema_history','checksum','FALSE','FALSE'),
+('flyway_schema_history','installed_by','FALSE','FALSE'),
+('flyway_schema_history','installed_on','FALSE','FALSE'),
+('flyway_schema_history','execution_time','FALSE','FALSE'),
+('flyway_schema_history','success','FALSE','FALSE'),
+('intervention','id','FALSE','TRUE'),
+('intervention','dynamic_framework_contract_id','FALSE','TRUE'),
+('intervention','created_at','FALSE','TRUE'),
+('intervention','title','FALSE','TRUE'),
+('intervention','description','FALSE','TRUE'),
+('intervention','incoming_referral_distribution_email','TRUE','TRUE'),
+('intervention_catalogue','id','FALSE','TRUE'),
+('intervention_catalogue','name','FALSE','TRUE'),
+('intervention_catalogue','int_type','FALSE','TRUE'),
+('intervention_catalogue','short_description','FALSE','TRUE'),
+('intervention_catalogue','long_description','FALSE','TRUE'),
+('intervention_catalogue','topic','FALSE','TRUE'),
+('intervention_catalogue','session_detail','FALSE','TRUE'),
+('intervention_catalogue','commencement_date','FALSE','TRUE'),
+('intervention_catalogue','termination_date','FALSE','TRUE'),
+('intervention_catalogue','created','FALSE','TRUE'),
+('intervention_catalogue','created_by','FALSE','TRUE'),
+('intervention_catalogue','last_modified','FALSE','TRUE'),
+('intervention_catalogue','last_modified_by','FALSE','TRUE'),
+('intervention_catalogue','reason_for_referral','FALSE','TRUE'),
+('intervention_catalogue_map','intervention_catalogue_id','FALSE','TRUE'),
+('intervention_catalogue_map','intervention_id','FALSE','TRUE'),
+('metadata','table_name','FALSE','FALSE'),
+('metadata','column_name','FALSE','FALSE'),
+('metadata','sensitive','FALSE','FALSE'),
+('metadata','domain_data','FALSE','FALSE'),
+('nps_region','id','FALSE','TRUE'),
+('nps_region','name','FALSE','TRUE'),
+('offence_type_ref','id','FALSE','TRUE'),
+('offence_type_ref','name','FALSE','TRUE'),
+('pcc_region','id','FALSE','TRUE'),
+('pcc_region','name','FALSE','TRUE'),
+('pcc_region','nps_region_id','FALSE','TRUE'),
+('personal_eligibility','id','FALSE','TRUE'),
+('personal_eligibility','intervention_id','FALSE','TRUE'),
+('personal_eligibility','min_age','FALSE','TRUE'),
+('personal_eligibility','max_age','FALSE','TRUE'),
+('personal_eligibility','males','FALSE','TRUE'),
+('personal_eligibility','females','FALSE','TRUE'),
+('possible_outcome','id','FALSE','TRUE'),
+('possible_outcome','intervention_id','FALSE','TRUE'),
+('possible_outcome','outcome','FALSE','TRUE'),
+('risk_consideration','id','FALSE','TRUE'),
```

Comment:
Would any of the risk stuff be considered sensitive data?

Reply:
As it's generic in the catalogue I'd say not, once it relates to a specific individual I would say yes.  @rgforsyth